images/harbor/main.tf

```diff
@@ -29,13 +30,17 @@ locals {
   repositories = {
     for k, v in local.components : k => "${var.target_repository}-${k}"
   }
+
+  // Update this when PostgreSQL versions for the latest Harbor DB release change
+  postgres_versions = "13 14"
```

Comment:
Could we define these as a list? and either pass the list through or unpack it before passing into ./config?

Reply:
I like that a lot better. We can certainly do that